media/base/status.h

```diff
@@ -58,6 +61,7 @@ struct SecondArgType<R(A1, A2)> {
   using Type = A2;
 };
 
+#if !defined(STARBOARD)
 union UKMPackHelper {
```

Comment:
Just curious on what the error is if UKMPackHelper is not excluded.

Reply:
On line 66/70, `__attribute__((packed))` is a gcc attribute and won't build with the MVSC toolchain on XB1 and Windows. It seems we don't need it, so I didn't translate it into an MVSC equivalent version.

Specific error on win-win32:
```
C:\Users\austinosagie\cobalt\cobalt_src\media/base/status.h(70): error C2059: syntax error: '('
C:\Users\austinosagie\cobalt\cobalt_src\media/base/status.h(70): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
C:\Users\austinosagie\cobalt\cobalt_src\media/base/status.h(70): error C2059: syntax error: ')'
C:\Users\austinosagie\cobalt\cobalt_src\media/base/status.h(70): error C2059: syntax error: ')'
C:\Users\austinosagie\cobalt\cobalt_src\media/base/status.h(70): error C2208: 'media::internal::UKMPackHelper::bits': no members defined using this type
```